webui/react/src/pages/HGICluster.tsx

```diff
@@ -55,8 +121,23 @@ const HGICluster: React.FC = () => {
     return <Message title="No Slots available" />;
   }
 
+  const viewOptions = (
+    <div className={css.viewOptions}>
+      <Link
+        className={selectedView === View.Grid ? css.active : ''}
+        onClick={setCardView}>
+        <Icon name="grid" size="large" title="Card View" />
+      </Link>
+      <Link
+        className={selectedView === View.List ? css.active : ''}
+        onClick={setTableView}>
+        <Icon name="list" size="large" title="Table View" />
+      </Link>
+    </div>
+  );
+
   return (
-    <Page id="cluster" title="HGI Cluster">
+    <Page className={css.base} id="cluster" title="HGI Cluster">
```

Comment:
**suggestion:** Just plain `Cluster` instead of `HGI Cluster` since HGI is an internal label?